apps/studio/lib/helpers.test.ts

```diff
@@ -0,0 +1,346 @@
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest'
+import {
+  tryParseJson,
+  minifyJSON,
+  prettifyJSON,
+  removeJSONTrailingComma,
+  timeout,
+  getURL,
+  makeRandomString,
+  pluckObjectFields,
+  tryParseInt,
+  propsAreEqual,
+  formatBytes,
+  snakeToCamel,
+  copyToClipboard,
+  detectBrowser,
+  detectOS,
+  pluralize,
+  isValidHttpUrl,
+  removeCommentsFromSql,
+  getSemanticVersion,
+  getDatabaseMajorVersion,
+  getDistanceLatLonKM,
+  formatCurrency,
+} from './helpers'
+
+describe('tryParseJson', () => {
```

Comment:
just a nit: these JSON helpers should also test the `catch` blocks for unhappy paths too